hdfs-scheduler/src/test/java/org/apache/mesos/hdfs/TestHdfsState.java

```diff
@@ -0,0 +1,325 @@
+package org.apache.mesos.hdfs;
+
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.security.SecureRandom;
+import java.util.concurrent.ExecutionException;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import org.apache.mesos.hdfs.scheduler.Task;
+import org.apache.mesos.hdfs.state.HdfsState;
+import org.apache.mesos.hdfs.util.HDFSConstants;
+import org.apache.mesos.hdfs.util.StatusFactory;
+import org.apache.mesos.Protos.CommandInfo;
+import org.apache.mesos.Protos.ExecutorID;
+import org.apache.mesos.Protos.ExecutorInfo;
+import org.apache.mesos.Protos.FrameworkID;
+import org.apache.mesos.Protos.Label;
+import org.apache.mesos.Protos.Labels;
+import org.apache.mesos.Protos.Offer;
+import org.apache.mesos.Protos.OfferID;
+import org.apache.mesos.Protos.Resource;
+import org.apache.mesos.Protos.SlaveID;
+import org.apache.mesos.Protos.TaskID;
+import org.apache.mesos.Protos.TaskState;
+import org.apache.mesos.Protos.TaskStatus;
+import org.apache.mesos.Protos.Value;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class TestHdfsState {
+  private final Injector injector = Guice.createInjector(new TestSchedulerModule());
+  private final Log log = LogFactory.getLog(TestHdfsState.class);
+  private SecureRandom random = new SecureRandom();
```

Comment:
I don't see any value in using random here... I realize the IDs are pseudo random in production, but mimicking that here doesn't add anything and actually detracts.